lively.morphic/rendering/renderer.js

```diff
@@ -1796,13 +1796,14 @@ export default class Renderer {
 
     const textNode = node.querySelector(`#${morph.id}textLayer`);
     const prevParent = textNode.parentNode;
+    textNode.remove();
     this.placeholder.className = 'Text';
-    this.placeholder.appendChild(textNode);
     textNode.style.width = 'max-content';
     textNode.style.height = 'max-content';
     textNode.style.position = 'static';
     if (morph.fixedWidth) textNode.style.width = morph.width + 'px';
     if (morph.fixedHeight) textNode.style.height = morph.height + 'px';
+    this.placeholder.appendChild(textNode);
```

Comment:
I do not understand this change. According to the commit message, this should make sure that the node gets removed from the placeholder. However, the removal of the placeholder is what happens in line 1811 I'd think?

Explicitly calling remove is also okay I guess, but as I understood it, both versions should behave exactly the same as appending a node that is mounted somewhere else results in the node being removed there.